README.md

```diff
@@ -66,6 +66,34 @@ new SwupFormsPlugin({
 });
 ```
 
+## Inline Forms
+
+If you give a form an additional attribute `[data-swup-inline-form]`, swup will:
+
+- update only that form when being submitted, ignoring the default `containers`.
+- scroll back to the beginning of the form
+- scope animations to the form itself
+
+> **Note** If you mark a form as an inline form, the form **must have an `id` attribute**
+
+### Example
+
+**HTML**
+```html
+<form id="form-1" data-swup-form data-swup-inline-form method="POST">
+  <input name="test"></input> <input type="submit"></input>
+</form>
+```
+**CSS**
+```css
+#form-1.is-changing {
```

Comment:
I'd suggest making it a bit easier for newcomers and using the "traditional" animation classes here so it's immediately familiar to people having read the Getting Started guide: e.g. `.transition-form.is-changing`. It feels a bit like I'm having to learn something new here with the `#id.is-changing` syntax.